src/pkgs.jl

```diff
@@ -389,7 +389,7 @@ function manifest_file(project_file)
         mfile = @static if isdefined(Base, :TOMLCache)
             Base.project_file_manifest_path(project_file, Base.TOMLCache())
         else
-            Base.project_file_manifest_path(project_file)
+            isfile(project_file) ? Base.project_file_manifest_path(project_file) : nothing
```

Comment:
It would be better to also use this check in the `TOMLCache` path, something like
```julia
mfile = !isfile(project_file) ? nothing : @static if isdefined...
```